tests/e2e/website-learning/websiteLearning.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from "@playwright/test";
+import { WebsiteLearningPage } from "../utils/page-objects/websiteLearningPage";
+
+test.use({ storageState: "tests/e2e/.auth/user.json" });
+
+test.describe("Website Learning UI Smoke Tests", () => {
+  let websiteLearningPage: WebsiteLearningPage;
+
+  test.beforeEach(async ({ page }) => {
+    websiteLearningPage = new WebsiteLearningPage(page);
+    try {
+      await websiteLearningPage.navigateToKnowledgeSettings();
+      await page.waitForLoadState("networkidle", { timeout: 10000 });
+    } catch (error) {
+      console.log("Navigation failed, retrying...", error);
+      await websiteLearningPage.navigateToKnowledgeSettings();
+      await page.waitForLoadState("domcontentloaded", { timeout: 30000 });
+    }
+  });
+
+  test("displays main website learning interface", async () => {
+    await websiteLearningPage.expectWebsiteLearningSection();
+    await websiteLearningPage.expectAddWebsiteButton();
+  });
+
+  test("shows add website form on button click", async () => {
+    await websiteLearningPage.clickAddWebsite();
+    await websiteLearningPage.expectAddWebsiteForm();
+  });
```

Comment:
I think we can merge these

```suggestion
  test("displays the website learning section and add website form", async () => {
    await websiteLearningPage.expectWebsiteLearningSection();
    await websiteLearningPage.expectAddWebsiteButton();

    await websiteLearningPage.clickAddWebsite();
    await websiteLearningPage.expectAddWebsiteForm();
  });
```